src/pybind/mgr/dashboard/frontend/src/app/shared/constants/app.constants.ts

```diff
@@ -150,7 +150,7 @@ export class ActionLabelsI18n {
 
     this.IMPORT = $localize`Import`;
 
-    this.MIGRATE = $localize`Migrate to Multi-Site`;
+    this.MIGRATE = $localize`Migrate to Multi-site`;
```

Comment:
I don't know why this says `Migrate to Multi-site` in the first place. It should have been just `Migrate` and rest of it should be have been done in the multi-site component. But let's deal with it later!

Reply:
From what I am seeing in the code,  this is used only for the action of "rgw migration".
Make sense to make it only migrate like other action verbs..I can take this in separate PR as code refactor..

https://github.com/ceph/ceph/blob/d3256c484136a1b32b79a904861f681a9248ba3c/src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts#L223-L228


https://github.com/ceph/ceph/blob/d3256c484136a1b32b79a904861f681a9248ba3c/src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-migrate/rgw-multisite-migrate.component.ts#L180-L186

Reply:
I have created a new tracker https://tracker.ceph.com/issues/64152 to fix it!